frontend/src/components/Documentation.vue

```diff
@@ -255,7 +255,7 @@ export default {
   data() {
     return {
       multipleColors,
-      ftpUrl: 'metabolicatlas.org',
+      ftpUrl: 'ftp.metabolicatlas.org',
```

Comment:
I received a `Your connection is not private` error when trying to access https://ftp.metabolicatlas.org/. After accepting the risk, it just directed me to the metatlas homepage. Maybe the DNS settings has not taken the effect yet?

Reply:
> @mihai-sysbio When I used the default `macOS` method (Finder -> Connect to Server -> Connect As Guest), it doesn't seem to be able to make the connection. Does that work for you?

It worked for me, but I think I forgot to purge the cache.

Reply:
> I received a `Your connection is not private` error when trying to access https://ftp.metabolicatlas.org/. After accepting the risk, it just directed me to the metatlas homepage. Maybe the DNS settings has not taken the effect yet?

I tested this and it worked for me, the TTL was 2 min.

Reply:
I see. It works with either FTP client and in Mac Finder when connected as Guest user.

Reply:
@nanjiangshu Good that you can confirm it works for you. It still doesn't connect for me but maybe I have some local settings that's preventing it. I consider this solved now since both of you can verify it.